setup.cfg

```diff
@@ -1,3 +1,53 @@
+[metadata]
+name = optuna-dashboard
+version = attr: optuna_dashboard.version.__version__
+
+url = https://github.com/optuna/optuna-dashboard
+author = Masashi Shibata
+author_email = m.shibata1020@gmail.com
+description = Real-time dashboard for Optuna.
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = MIT License
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3 :: Only
+    Intended Audience :: Science/Research
+
+[options]
+python_requires = >=3.6
+include_package_data = True
+packages = find:
+install_requires =
+    optuna>=2.4
+    bottle
+    typing-extensions;python_version<'3.8'
+
+[options.extras_require]
+lint =
+    black
+    mypy
+    flake8
+
+[options.entry_points]
+console_scripts =
+    optuna-dashboard = optuna_dashboard.cli:main
+
+[options.package_data]
+optuna_dashboard = public/*
```

Comment:
memo: This seems to be ignored 🤔

Reply:
This is due to the following line.
https://github.com/pypa/setuptools/blob/fb7ab81a3d080422687bad71f9ae9d36eeefbee2/setuptools/command/sdist.py#L114-L121

(`build_py.data_files` holds the files in `public/bundle.js` and so on, but ignored because include_package_data is true.)